app_dart/lib/src/service/luci_build_service/user_data.dart

```diff
@@ -138,7 +138,7 @@ final class _PresubmitUserData extends PresubmitUserData {
   @override
   final int checkRunId;
 
-  @JsonKey(name: 'check_suite_id')
+  @JsonKey(name: 'check_suite_id', defaultValue: 0)
```

Comment:
File an issue and add A TODO to remove this in a couple of days once all outstanding messages are ack'ed successfully.